### DISCUSSION TITLE
args options optimize for v1.0

### DISCUSSION THREAD

**Discussion by yxxhero** [helmfile/helmfile#381] [Ideas]
args options optimize for v1.0

**Comment by yxxhero** [comment]
--args splite into 
* --upgrade-args
* --list-args
* --uninstall-args

Make args configuration more refined and controllable


@mumoshu ping

**Comment by frank-berlin** [comment]
`helmfile template --args "--show-only templates/xxxx.yaml"`
is a use case address for v1
could be: `--template-args`
or add flag
`helmfile template --show-only templates/xxxx.yaml`
`helmfile template -s templates/xxxx.yaml`

**Comment by yxxhero** [comment]
@mumoshu WDYT?

**Comment by yxxhero** [comment]
https://github.com/helmfile/helmfile/pull/959
